assets/css/main.css

```diff
@@ -0,0 +1,106 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Lato:wght@100&family=Open+Sans:wght@300&family=Poppins&family=Roboto:ital,wght@0,700;1,100&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Arial", "Helvetica", sans-serif;
+  font-size: 12px;
+}
+
+/* variables */
+:root {
+  --dark-grey: #3c3a39;
+  --black: #1c1a19;
+  --main-white: #fff;
+}
+
+/* header section */
+#header {
+  width: 100%;
+  background-color: var(--dark-grey);
+  height: 92px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 20px;
+}
+
+#header a {
+  text-decoration: none;
+  color: var(--main-white);
+  font-family: "Roboto", "Helvetica", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.nav_icon {
+  cursor: pointer;
+}
+
+/* end of header
+section */
+
+/* headline section */
+#headline {
+  height: 590px;
+  width: 100%;
+  background-color: var(--black);
+  padding: 24px;
+  position: relative;
```

Comment:
- [ ] Please use `flexbox` to align the whole section content like the project requirement.

![image](https://user-images.githubusercontent.com/68796232/158793851-b0810ac7-3db5-42fd-b47b-89b78a3596fc.png)

That's all have a great day